src/lib/RichText/Converter/ListLineBreaks.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\FieldTypeRichText\RichText\Converter;
+
+use EzSystems\EzPlatformRichText\eZ\RichText\Converter;
+use DOMDocument;
+use DOMXPath;
+
+/**
+ * Class ListLineBreaks.
+ *
+ * Processes lists below <literallayout> tag.
+ */
+class ListLineBreaks implements Converter
+{
+    /**
+     * For all <itemizedList> and <orderedList> below <literallayout>, move the list after the <literallayout>, so that it is not inside it.
+     *
+     * @param \DOMDocument $document
+     *
+     * @return \DOMDocument
+     */
+    public function convert(DOMDocument $document)
+    {
+        $xpath = new DOMXPath($document);
+        $xpathExpression = '//ns:literallayout [descendant::ns:orderedlist|descendant::ns:itemizedlist]';
+        $ns = $document->documentElement ? $document->documentElement->namespaceURI ?: '' : '';
+        $xpath->registerNamespace('ns', $ns);
```

Comment:
We shouldn't register empty namespace.
```php
private const FALLBACK_NAMESPACE = 'http://docbook.org/ns/docbook';
```
```suggestion
        $xpath->registerNamespace(
            'ns',
            null !== $document->documentElement && !empty($document->documentElement->namespaceURI)
                ? $document->documentElement->namespaceURI
                : self::FALLBACK_NAMESPACE
        );
```

Reply:
Fixed as suggested in 295b2127692c3b2b2aaa26c0c9310cb3c916a2f4